tests/flytekit/unit/models/test_workflow_closure.py

```diff
@@ -58,6 +61,7 @@ def test_workflow_closure():
             {},
             {},
         ),
+        extended_resources=_core_task.ExtendedResources(gpu_accelerator=NvidiaTeslaT4.to_flyte_idl()),
```

Comment:
why are we changing this?

Reply:
Probably not super necessary, but just wanted to make sure that this test covers the full `TaskTemplate` object.